share/lib/python/neuron/rxdtests/run_all.py

```diff
@@ -33,7 +33,7 @@ def test(files):
 
         
         try:
-            outp = subprocess.check_output(['python%i' % sys.version_info.major, 'do_test.py', os.path.join('tests', f), output_file] )
+            outp = subprocess.check_output([sys.executable, 'do_test.py', os.path.join('tests', f), output_file] )
```

Comment:
@adamjhn  @nrnhines : this was needed because on my machine there was no `python2` but `python2.7`. In this case, I thought `sys.executable` is more robust way to get python command used to launch run_all.py itself.